server/src/main/java/org/apache/iotdb/db/qp/physical/crud/DeletePartitionPlan.java

```diff
@@ -41,6 +41,11 @@ public DeletePartitionPlan(String storageGroupName, Set<Long> partitionId) {
     return null;
   }
 
+  @Override
+  public List<String> getPathsStrings() {
+    return null;
```

Comment:
It is not a good idea to return a null collection, use `Collections.emptyList()` instead.

Reply:
I'll change it. Thanks for your suggestion.